appoptics_apm.gemspec

```diff
@@ -43,7 +46,9 @@ Automatic tracing and metrics for Ruby applications. Get started at appoptics.co
 
   s.extensions = ['ext/oboe_metal/extconf.rb'] unless defined?(JRUBY_VERSION)
 
-  s.add_runtime_dependency('json')
+  # this still gives a warning, would have to be pinned to a minor version
+  # but that is not necessary and may restrict other gems
+  s.add_runtime_dependency('json', '>= 0.4.0')
```

Comment:
Hmm that's too bad since json 0.4.0 is quite old and a quick search shows CVE vulnerability before version 1.5.x.  But it sounds like our agent dependency could actually "restrict other gems" in the instrumented application, meaning cause some sort of dependency conflict when customers install us?

Reply:
let me see if why we need json or if fixing to a major version is an option

Reply:
Looks like `s.add_runtime_dependency('json', '~> 2.0')` works, that is broad enough and excludes breaking changes from a new major version